src/commands/BootstrapCommand.js

```diff
@@ -46,13 +46,19 @@ export default class BootstrapCommand extends Command {
   }
 
   initialize(callback) {
-    const { registry, npmClient } = this.options;
+    const { registry, npmClient, npmClientArgs } = this.options;
```

Comment:
Specifically, using this block starting at line 12:

```js
export function handler(argv) {
  return new BootstrapCommand([...argv.npmClientArgs], argv).run();
}

export const command = "bootstrap [npm-client-args..]";
```

This pattern is used in `lerna exec` (and `run`): https://github.com/lerna/lerna/blob/master/src/commands/ExecCommand.js#L8

Thus you can alias a collection of all unparsed positional arguments, as well as unparsed remainders behind `--`.

```
lerna bootstrap --scope foo -- --bar
lerna bootstrap --hoist -- --no-optional
```

It should still be retrievable from `this.options.npmClientArgs`, but it's also always present as an array (which might be empty).